.github/container/Dockerfile.base

```diff
@@ -17,30 +17,37 @@ ARG DEST_MANIFEST_DIR
 ## Install Python and essential tools
 ###############################################################################
 
-RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive \
-    TZ=America/Los_Angeles \
-    apt-get install -y \
-        build-essential \
-        checkinstall \
-        clang \
-        cmake \
-        curl \
-        git \
-        lld \
-        vim \
-        bat \
-        curl \
-        git \
-        gnupg \
-        rsync \
-        python-is-python3 \
-        python3-pip \
-        liblzma-dev \
-        wget \
-    && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN <<"EOF" bash -ex
+export DEBIAN_FRONTEND=noninteractive
+export TZ=America/Los_Angeles
+apt-get update
+apt-get install -y curl
+
+# Repo for newer nsight versions
+curl -o /usr/share/keyrings/nvidia.pub https://developer.download.nvidia.com/devtools/repos/ubuntu2204/$(dpkg --print-architecture)/nvidia.pub
```

Comment:
nit: I know docker build has some [automatically populated build args](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope), do you know if any of these can replace the `dpkg` invocation?

Reply:
`TARGETARCH` apparently does. Empirically we do know it's `amd64` and `arm64` on two architectures that we care about. However, I'm not able to find a formal specification of the values that it can take.

Reply:
IMO it's better to get a string from the package management machinery if it's being fed into the package management machinery, given that it's not clear whether the conventions match.